### DISCUSSION TITLE
Not able to run FundMe.sol correctly

### DISCUSSION THREAD

**Discussion by GH023** [smartcontractkit/full-blockchain-solidity-course-py#167] [Q&A] [answered]
Hello,

When I deploy the FundMe.sol file, I get the error below. This error occurs everytime when the value is not empty (ie. anything other than 0 wei). It runs fine if the value is set to 0 wei.

![github error fundme](https://user-images.githubusercontent.com/92254807/136694069-7c34cc3f-6a0d-4e3c-afc7-dae752f1682d.PNG)

I have 18 ETH in my account and using the Rinkeby test network. The account is in the Rinkeby network. I am using the original file from Patrick from here https://github.com/PatrickAlphaC/fund_me/blob/main/FundMe.sol and using compiler v0.6.6 on remix web ide.

**Comment by PatrickAlphaC** [comment, marked as answer]
You can't deploy a contract and send wei to it at the same time! Please always have that set to 0 wei when you're deploying a smart contract.
